backend/song/tests/test_views.py

```diff
@@ -0,0 +1,272 @@
+import pytest
+
+from django.urls import reverse
+
+from rest_framework.status import (
+    HTTP_200_OK,
+    HTTP_201_CREATED,
+    HTTP_204_NO_CONTENT,
+    HTTP_403_FORBIDDEN,
+)
+
+from backend.song.models import Artist
+
+
+pytestmark = pytest.mark.django_db
+
+
+class TestCategoryView:
+
+    def test_create_category(self, auth, create_category):
+
+        assert create_category.status_code == HTTP_201_CREATED
+        assert create_category
```

Comment:
what do you think about testing if the response have all field we are expecting it to have? for the creation and especially get method?  this could help us to avoid unexpected errors when integrating with the front end...

Reply:
@tadomikikuto-bit? 🤔